modules/ROOT/pages/update-streams-switching.adoc

```diff
@@ -0,0 +1,10 @@
+
+= Switching to a Different Stream
+
+In order to switch between the different streams of Fedora CoreOS (FCOS) a user can leverage the `rpm-ostree rebase` command. The following three commands will switch the respective systems to each of the three streams that are available and reboot the system to boot into the new stream.
+
+- `sudo rpm-ostree rebase fedora/x86_64/coreos/stable --reboot`
+- `sudo rpm-ostree rebase fedora/x86_64/coreos/testing --reboot`
+- `sudo rpm-ostree rebase fedora/x86_64/coreos/next --reboot`
```

Comment:
I think you can collapse this using a
```
# Available streams: "stable", "testing", and "next".
STREAM="stable" 
```

Also, there should be a `systemctl stop zincati.service` before this.

Reply:
ok I added it